blueprints/in-repo-engine/files/lib/__name__/index.js

```diff
@@ -0,0 +1,8 @@
+/*jshint node:true*/
+module.exports = {
+  name: '<%= dasherizedModuleName %>',
+
+  isDevelopingAddon: function() {
+    return true;
+  }
+};
```

Comment:
@trentmwillis - Awesome, thanks for confirming this.  I think that in-repo-addons inside an addon should generally work without extra fuss (likely a bug in Ember CLI itself).